modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/resource/OpenAPIResourceImpl.java

```diff
@@ -1138,6 +1138,9 @@ private Schema<Object> _addSchema(DTOProperty dtoProperty) {
 				if (type.equals("Boolean")) {
 					schema.setType("boolean");
 				}
+				else if (type.equals("Clob")) {
+					schema.setType("string");
+				}
```

Comment:
@thallescezar I believe we can solve this on our side at [this method](https://github.com/MarinhoFeliphe/liferay-portal/blob/master/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/openapi/v1_0/ObjectEntryOpenAPIResourceImpl.java#L149) so we would not need to modify this class from Headless team side,

I believe the we can create another conditional by passing the String type instead of CLOB [here](https://github.com/MarinhoFeliphe/liferay-portal/blob/master/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/openapi/v1_0/ObjectEntryOpenAPIResourceImpl.java#L225), what do you think?